packages/ord-connect/src/hooks/useSignMessage.tsx

```diff
@@ -26,6 +32,7 @@ export function useSignMessage(): {
           wallet,
           message,
           network,
+          walletAddresses,
```

Comment:
Instead of passing `walletAddresses`, should pass `format`. Maybe we should also check that `address` being passed matches either `walletAddresses.ordinals` or `walletAddresses.payments`?

We will also need a helper function to get the format based on the selected wallet address

Reply:
created a function `getAddressFormat` to get the address format

Reply:
added a comment about this, i don't think we will need that function

Reply:
updated